src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -390,7 +376,13 @@
                         (fn [ws]
                           (let [ws-tags (set (get-in ws [:workspace :attributes :tag:tags :items]))]
                             (every? (partial contains? ws-tags) selected-tags))))]
-       (filter (apply every-pred tag-filter checkbox-filters) workspaces)))})
+       (filter (apply every-pred tag-filter checkbox-filters) workspaces)))
+   :-show-request-access-modal
+   (fn [_ workspace-id auth-domains]
```

Comment:
Since this is ignoring all of the React internals, it might make more sense for this to be an external function.